cropper/src/main/java/com/canhub/cropper/CropImageOptions.kt

```diff
@@ -343,6 +343,14 @@ open class CropImageOptions : Parcelable {
     @JvmField
     var cropperLabelText: String? = ""
 
+    /** Crop Image background color **/
+    @JvmField
+    var activityBackgroundColor: Int = -1
+
+    /** Toolbar color **/
+    @JvmField
+    var toolbarColor: Int = -1
```

Comment:
We can add a default value, so we can avoid the logic to check `-1`

Reply:
Totally valid point.

Can we get these values here? as default? 🤔

Reply:
I just checked this.

We can set default Color WHITE to activity background.

But for toolbar I'm not sure how to fetch the primaryColor from the app themes file.

I found 2 solutions here.
[Solution 1](https://stackoverflow.com/questions/28489509/how-can-i-get-the-primary-color-from-my-app-theme)
[Solution 2](https://stackoverflow.com/questions/27611173/how-to-get-accent-color-programmatically)

Unfortunately they both require `context` which is not available in `CropImageOptions`.

I also tried `Resources.getSystem().getColor()` method. It has 2 parameters. The first one takes only an id of a color we have defined. So we cannot fetch the `colorPrimary` attribute using this.

We could also manually set the toolbar color to the primary color stated in the themes file which is currently `purpleDark` but if in the future this attribute is updated in the themes file it might not be updated for the toolbar

Reply:
I will let you decide what makes more sense here. Any solution is good for me

just think about what is easy for our users and for maintenance.

Reply:
Is it ok to keep a default value of -1 only for toolbar here? I can't think of any other option that can fetch the primaryColor from the theme file.

So if they don't set the toolbar color it will use the default primary color set in the theme file.

Reply:
yes